src/osip-monitoring-view/src/main/java/edu/kit/pse/osip/monitoring/view/dashboard/MonitoringMainWindow.java

```diff
@@ -1,36 +1,138 @@
 package edu.kit.pse.osip.monitoring.view.dashboard;
 
+import edu.kit.pse.osip.core.model.base.ProductionSite;
 import edu.kit.pse.osip.core.model.base.TankSelector;
+import edu.kit.pse.osip.core.utils.language.Translator;
+import java.util.EnumMap;
+import javafx.geometry.Insets;
+import javafx.scene.Scene;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.HBox;
+import javafx.stage.Stage;
 
 /**
  * The entry point for the monitoring view. It shows all sensor datas graphically.
+ * 
+ * @author Martin Armbruster
+ * @version 1.1
  */
-public class MonitoringMainWindow {
-    public MixTankVisualization mixTank;
-    public TankVisualization tanks;
-    public Light light;
-    public LoggingConsole log;
-    public MonitoringMenu menu;
+class MonitoringMainWindow {
```

Comment:
Das Log Feld skaliert momentan nicht in der Höhe mit. Finde, die Tanks sind doch das eigentlich wichtige. Also sollten nicht nur die Tanks kleiner werden, sondern auch dieses Feld